core/src/common/dtls_abstraction_cyassl.c

```diff
@@ -268,14 +300,34 @@ static void SetupNewSession(int index, NetworkAddress * networkAddress, bool cli
         session->Context =  CyaSSL_CTX_new(CyaDTLSv1_2_server_method());
     if (session->Context)
     {
-        CyaSSL_CTX_set_cipher_list(session->Context, "ECDHE-ECDSA-AES128-CCM-8:ECDHE-ECDSA-AES128-SHA256:PSK-AES128-CCM-8:PSK-AES128-CBC-SHA256");
+
         if (certificate)
         {
             int format = SSL_FILETYPE_PEM;
             if (certificateFormat == CertificateFormat_ASN1)
                 format = SSL_FILETYPE_ASN1 ;
-            CyaSSL_CTX_use_certificate_buffer(session->Context, certificate, certificateLength, format);
-            CyaSSL_CTX_use_PrivateKey_buffer(session->Context, certificate, certificateLength, format);
+
```

Comment:
We prefer spaces rather than tabs in Awa, to remain consistent with the overall project.